Composer/plugins/samples/assets/projects/CustomAction/schemas/sdk.schema

```diff
@@ -0,0 +1,8901 @@
+{
```

Comment:
this should be an update to the default sdk.schema, instead of creating a new one.